modules/images-other-jenkins-memory.adoc

```diff
@@ -13,6 +13,10 @@ than a total of `512 MiB` of memory. If they require more memory, the container
 halts. It is therefore highly recommended that pipelines run external commands
 in a agent container wherever possible.
 
+And if `Project` quotas allow for it, see recommendations from the Jenkins
+documentation on what a Jenkins master should have from a memory perspective.
+Those recommendations proscribe to allocate even more memory for the Jenkins master.
+
```

Comment:
Suggested change:
"If `Project` quotas allow for it, you can refer to the link:https://jenkins.io/doc/pipeline/tour/getting-started/[Jenkins documentation] for the memory requirements a Jenkins master should have. 
Those recommendations proscribe the allocation of more memory for the Jenkins master."

I'm not super clear on this one. If the Jenkins recommendations forbid the allocation of more memory for the Jenkins master, then should we refer the user to their docs or would it be better for us to clearly state the requirements?

Reply:
Not from your PR @gabemontero , but we could maybe also fix the line before that:
" It is therefore highly recommended that pipelines run external commands
 in a agent container wherever possible.		 in a agent container wherever possible."

in a**n** agent container

Reply:
@bmcelvee - some current specifics I don't want to explicitly cite in the docs ... last I looked, 2GiB of allocation to the JVM (meaning a 4 GiB pod memory limit) is the current recommendation out of jenkins doc ... that historically has proven untenable for starter, given the project quotas they have for memory in starter, hence the 1 GiB in the template ... the reality is you can bring Jenkins up, do some limited stuff, but it can get slow quick, at 1 GiB total mem / 512 MiB JVM heap for jenkins.

FYI that link you found https://jenkins.io/doc/pipeline/tour/getting-started/ is not the real life recommendation.

See https://support.cloudbees.com/hc/en-us/articles/204859670-Java-Heap-settings-best-practice

Given all that, I want to recommend that they look at the Jenkins doc and override what our templates have for anything *real*.

With all that, can you proscribe some text accordingly?

Reply:
I'll fix the grammar item you found @akram